index.js

```diff
@@ -116,12 +129,20 @@ class LedgerBridgeKeyring extends EventEmitter {
             } else {
               address = this._addressFromIndex(pathBase, i)
             }
+
+            console.log("[LedgerBridgeKeyring][addAccounts] Adding account: ", address)
+
             this.accounts.push(address)
             this.page = 0
           }
+          this._sendMessage({ action: 'ledger-close-bridge' }, () => this.forgetDevice())
```

Comment:
I believe this call is responsible for the issue you encountered where you were required to lock then unlock the device before seeing your account.

`forgetDevice` clears the keyring, removing the account altogether. Nothing will show up in the account menu after this is called.

Reply:
Bingo!